To: vim-dev@vim.org
Subject: Patch 7.3.015
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.015
Problem:    Test is using error message that no longer exists.
Solution:   Change E106 to E121. (Dominique Pelle)
Files:	    src/testdir/test49.vim


*** ../vim-7.3.014/src/testdir/test49.vim	2010-08-15 21:57:29.000000000 +0200
--- src/testdir/test49.vim	2010-09-29 16:52:06.000000000 +0200
***************
*** 1,6 ****
  " Vim script language tests
  " Author:	Servatius Brandt <Servatius.Brandt@fujitsu-siemens.com>
! " Last Change:	2006 Apr 28
  
  "-------------------------------------------------------------------------------
  " Test environment							    {{{1
--- 1,6 ----
  " Vim script language tests
  " Author:	Servatius Brandt <Servatius.Brandt@fujitsu-siemens.com>
! " Last Change:	2010 Sep 29
  
  "-------------------------------------------------------------------------------
  " Test environment							    {{{1
***************
*** 8711,8717 ****
  		endif
  	    else
  		if cmd == "let"
! 		    let match = MATCH(0, thrmsg, 'E106', "Unknown variable")
  		elseif cmd == "unlet"
  		    let match = MATCH(0, thrmsg, 'E108', "No such variable")
  		endif
--- 8711,8717 ----
  		endif
  	    else
  		if cmd == "let"
! 		    let match = MATCH(0, thrmsg, 'E121', "Undefined variable")
  		elseif cmd == "unlet"
  		    let match = MATCH(0, thrmsg, 'E108', "No such variable")
  		endif
*** ../vim-7.3.014/src/version.c	2010-09-29 15:50:14.000000000 +0200
--- src/version.c	2010-09-29 16:55:03.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     15,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
228. You spend Saturday night making the counter on your home page
     pass that 2000 mark.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
